### PR TITLE
[#86948890] remove github fork ribbon

### LINKED ISSUE #22: Remove Github fork ribbon
Cute but misleading because the platform components themselves are not open-source, e.g. Devicehub. Furthermore its causing a complication in our build right now. Lets just remove it.

Actually the best way to integrate the open-source component anyways is to have "modify docs" buttons through out the site so people can get a link from the site docs to a github `yaml` (etc.) file to edit immediately. Better context FTW. Angular etc. does this:

 
screen shot 2015-01-26 at 3 34 20 pm

### PR DESCRIPTION
**Pull request by stresslimit**
closes #22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,7 @@
 {
   "private": true,
-  "napa": {
-    "github-fork-ribbon-css": "simonwhitaker/github-fork-ribbon-css"
-  },
   "scripts": {
-    "start": "NODE_ENV=production node server",
+    "start": "NODE_ENV=production node-dev server",
```
Comment (jasonkuhrt) [inline package.json]: `npm start` is what the infrastructure will use to start the project during deployment. We should not be using `node-dev` in those settings. That's what `dev-server` task was for.

**Comment by jasonkuhrt** [comment]
Just that one fix then LGTM.

File: package.json (hunk 1 above)
Comment (stresslimit) [inline package.json]: ok but without node-dev it doesn't work for me.

Comment (jasonkuhrt) [inline package.json]: How so?

**Pull request closed without merging**